examples/demo-rollup/stf/src/hooks_impl.rs

```diff
@@ -13,24 +13,31 @@ use crate::runtime::Runtime;
 
 impl<C: Context, Da: DaSpec> TxHooks for Runtime<C, Da> {
     type Context = C;
+    type PreArg = ();
+    type PreResult = C::Address;
+    type PostArg = ();
+    type PostResult = ();
 
     fn pre_dispatch_tx_hook(
         &self,
         tx: &Transaction<Self::Context>,
         working_set: &mut WorkingSet<C>,
-    ) -> anyhow::Result<<Self::Context as Spec>::Address> {
+        _arg: (),
```

Comment:
Now we have unused parameters `()` lying around. 
And the `PostArg & PostResult` will always be `()`. What about going the other way we discussed and removing the `TxHooks` for `Accounts.`?

Reply:
Removed these to simplify the api